Tests/js/views/assets/ez-userprofileview-tests.js

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (C) eZ Systems AS. All rights reserved.
+ * For full copyright and license information view LICENSE file distributed with this source code.
+ */
+YUI.add('ez-userprofileview-tests', function (Y) {
+    var renderTest;
```

Comment:
the case where the view receives a valid `userAvatar` object is not covered. A test on the tap event is also missing and like in https://github.com/ezsystems/PlatformUIBundle/pull/513/files#r53947305, the `valueFn` of the `userMenuView` is not tested.